vignettes/admiral.Rmd

```diff
@@ -22,7 +22,10 @@ The main idea of `{admiral}` is that an ADaM dataset is built by a sequence of
 derivations. Each derivation adds one or more variables or parameters to the
 processed dataset. This modular approach makes it easy to adjust code by adding,
```

Comment:
Could we say records instead of parameters?

Reply:
done